README.md

```diff
@@ -62,6 +59,13 @@ Install this project with:
 
```

Comment:
- [ OPTIONAL ]  As pointed out by the previous reviewer, right now, your branch name is not descriptive enough fearful because you are using leader-board which is not giving enough information of what are doing with leaderboard so it would be nice if you make your branch name more descriptive such as `feature/leaderboard-final-touches` which is much more descriptive and gives proper information of what are you doing in this project